ush/fix_rrfs_locations.sh

```diff
@@ -10,7 +10,7 @@ elif [[ -d /cm ]] ; then
     FIX_RRFS_LOCATION="/to/do"
 elif [[ -d /ioddev_dell ]]; then
     PLATFORM=wcoss_dell_p3
-    FIX_RRFS_LOCATION="/gpfs/hps3/emc/meso/save/Ming.Hu/FIX_RRFS"
+    FIX_RRFS_LOCATION="/gpfs/dell6/emc/modeling/noscrub/emc.campara/FIX_RRFS"
```

Comment:
I have a PR before this and its changes are more complete than this. Could you approve that PR first and then remove this fix_rrfs_locations.sh change by this PR.?

Reply:
I have approved your PR and merge with this PR.